### ISSUE TITLE
Release Mink 1.7.2 or 1.8

### ISSUE THREAD

**Issue by klausi** [minkphp/Mink#725]
After #720 it would be cool to get a new Mink  release so that we can have a concrete version number in Drupal's composer.json.

Currently https://www.drupal.org/node/2808085 uses a git commit hash which is not ideal and people are hesitant to move forward with it.

Any timeline for releasing the next version?

**Comment by catch56** [comment]
Just bumping this. We're in the process of releasing Drupal 8.3.0 - the alpha is today. And this is the only dependency that's on a commit hash at the moment. Still a couple of weeks to fix that if a new Mink release was tagged.

**Comment by sanpii** [comment]
To release #747 it should be nice to have a new release :)

**Comment by anavarre** [comment]
Drupal 8.5.0-alpha1 is around the corner. @stof could you please tag a new release?

**Comment by Nek-** [comment]
This is becoming a serious issue. Ping @everzet

**Comment by Gadgetdude** [comment]
_Bumping this thread into 2019._

**Comment by shehi** [comment]
```symfony/css-selector``` release v4 and now this package is not installable with latest Symfony ecosystem. Please release new version.

**Comment by shehi** [comment]
@dunglas , are you able to make a release for this package? @everzet seems to be nowhere.

**Comment by greg-1-anderson** [comment]
Over in https://github.com/minkphp/MinkSelenium2Driver/issues/313, there is a discussion about the problems that Drupal is having as a result of using a dev-release, to work around the lack of a stable tag. We have the same problem with this repository: Drupal is still using 1.7.x-dev, and has been since Drupal 8.3.0 was released in 2017. When the 1.7.x branch alias in this repository is changed to 1.8.x, every stable Drupal tag since 8.3.0 will break.

We could begin to prepare to repair the looming damage if someone could tag a534fe7 as 1.7.2 or 1.8.0.

**Comment by aik099** [comment]
@greg-1-anderson , I agree with 1.7.2. If all tests are green, then we only need to prepare a changelog.

**Comment by greg-1-anderson** [comment]
@aik099: I converted a git log (minus trivial changes) into a rough changelog:

```
1.7.2 / 2019-09-XX
==================

Bug fixes:

* DriverInterface#getValue() can return null
* Modify response[Contains|NotContains] to use strpos vs preg_match
* Fix compatibility with PHP 7.2
* Allow Symfony 4.
* Optimize the splitting of XPath unions
* Allow pipes (|) in attributes by implementing better XPath parsing
* Auto-start session only upon 1st "->visit(...)" method call

Testsuite:

* Don't install PHPUnit 8.0
* Fix PHP 5.4 and PHP 5.5 builds
* Objects with __toString() can be asserted.
* Add testing on PHP 7.2
* Add compatibility with PHPUnit 6
* Remove the driver testsuite

Misc:

* Rename boolean type in lowercase
* Suggest chrome-mink-driver as fast JS-enabled alternative to selenium
* Use ! operator instead of === false
* Improved variable names and simplified quote logic
* Changed method visibility to private and variable name to camle case
```

Could make that a PR if you'd like.

**Comment by aik099** [comment]
@greg-1-anderson , yes, please send a PR where I'll indicate what parts of change log file should be adjusted.

**Comment by greg-1-anderson** [comment]
@aik099 Created https://github.com/minkphp/Mink/pull/786

**Comment by stof** [comment]
1.8.0 is finally released (as well as releases of BrowserKitDriver and Selenium2Driver).
